interface/patient_file/encounter/forms.php

```diff
@@ -35,8 +35,8 @@
 ?>
 
 <?php // include generic js support for graphing ?>
-<script type="text/javascript" src="<?php echo $modules_dir; ?>openflashchart/js/json/json2.js"></script>
-<script type="text/javascript" src="<?php echo $modules_dir; ?>openflashchart/js/swfobject.js"></script>
+    <script type="text/javascript" src="<?php echo $GLOBALS['web_root']?>/modules/openflashchart/js/json/json2.js"></script>
```

Comment:
While $modules_dir does not work, the $GLOBALS['modules_dir'] will do it for you.